Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM wordpress:5.0.3-fpm
```

Comment:
Is there a use the latest branch here, instead of pinning to a specific tag?

Reply:
So the idea is to match this with whatever we have in production as close as possible, so that we can reduce the number of things that we have to test on sandbox. I will update this, but lets keep it manual. Whenever we update production version, we will push a commit here as well.